docs/pages/dsCompositeIn/ds-compositeIn-L3-Low-Level-TestSpec.md

```diff
@@ -56,7 +56,8 @@ classDiagram
     testControl <|-- ut_raft : inherits
     class ut_raft{
     }
-    ut_raft <|-- L3_TestClasses : inherits
+    ut_raft <|-- dsCompositeInHelperClass : inherits
+    dsCompositeInHelperClass <|-- L3_TestClasses : inherits
```

Comment:
please change the note respect to yaml file think suite Navigator refers to testConfig file correct?  note  ut_raft "suite Navigator uses testSuite.yaml"

Reply:
done